### PR TITLE
add p9

### PR DESCRIPTION
**Pull request by sevki**

### CODE REVIEW

**Comment by sevki** [comment]
@hugelgupf could you have a look at this?

File: projects/p9/project.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+homepage: "https://github.com/hugelgupf/p9"
+language: go
+primary_contact: "harvey@googlegroups.com"
+auto_ccs:
+  - "s@sevki.org"
```
Comment (hugelgupf) [inline projects/p9/project.yaml:5]: pls add c@chrisko.ch

**Comment by Dor1s** [comment]
Thanks, we're accepting this project. Do you want to merge this or will you add fuzzer(s) first?

**Comment by sevki** [comment]
That's awesome news! I'll send a fuzzer :)

**Comment by sevki** [comment]
@Dor1s I followed the Go project setup guide, but there are some linking issues.

I've only ever built the fuzzer for this with go-fuzz-bulild and that just works. Is there additional documentation?

**Comment by inferno-chromium** [comment]
> @Dor1s I followed the Go project setup guide, but there are some linking issues.
> 
> I've only ever built the fuzzer for this with go-fuzz-bulild and that just works. Is there additional documentation?

You need the full path to fuzzer, e.g. see Max fix for syzkaller in https://github.com/google/oss-fuzz/commit/fa5b4dbd9aeca77a90aaa11417ef8153ac9e49d9#diff-3976d65d74ec2a9d4a8d326a8575cc01

**Comment by sevki** [comment]
thanks so much @inferno-chromium :)

**Comment by sevki** [comment]
this works locally. the install dependencies seems to be broken due to gh/azue infra problems :man_shrugging: @Dor1s @inferno-chromium thank you both for your help!

File: projects/p9/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2019 Google Inc.
```
Comment (Dor1s) [inline projects/p9/Dockerfile:1]: nit: 2020

File: projects/p9/build.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
+function compile_fuzzer {
```
Comment (Dor1s) [inline projects/p9/build.sh:1]: I suspect the presubmit is complaining about missing licence header here

**Comment by inferno-chromium** [comment]
@sevki - can you fix the comment nits that Max said and update to master and then push. We had a CI fix, so you need to pull +rebase to master before next push so that CI can test your cl properly.

**Comment by inferno-chromium** [comment]
@sevki - there is a startup crash on fuzzer, can you fix it. we don't archive the build if the fuzzer crashes instantly.
```
	_cgo_gotypes.go:53 +0x56
AddressSanitizer:DEADLYSIGNAL
=================================================================
==52==ERROR: AddressSanitizer: ABRT on unknown address 0x000000000034 (pc 0x0000005aee21 bp 0x10c000046cb0 sp 0x10c000046c98 T0)
SCARINESS: 10 (signal)
    #0 0x5aee21 in runtime.raise runtime/sys_linux_amd64.s:1

DEDUP_TOKEN: runtime.raise
AddressSanitizer can not provide additional info.
SUMMARY: AddressSanitizer: ABRT runtime/sys_linux_amd64.s:1 in runtime.raise
==52==ABORTING
MS: 0 ; base unit: 0000000000000000000000000000000000000000


artifact_prefix='./'; Test unit written to ./crash-da39a3ee5e6b4b0d3255bfef95601890afd80709
Base64: 
ERROR: 100% of fuzz targets seem to be broken. See the list above for a detailed information.
```

**Comment by sevki** [comment]
@inferno-chromium PTAL :smile:

**Pull request merged**